docs/_docs/reference/syntax.md

```diff
@@ -116,7 +116,7 @@ A `colon` token reads as the standard colon "`:`" but is generated instead of it
 is an alphanumeric identifier, a backticked identifier, or one of the tokens `this`, `super`, `new`, "`)`", and "`]`".
 
 ```
-colon         ::=  ':'    -- with side conditions explained above
+colon         ::=  ':' “with side conditions explained above”
```

Comment:
there are many such comments of this form throughout the document, it's probably better to leave as it is.

Reply:
```suggestion
colon         ::=  ':' -- with side conditions explained above
```